### ISSUE TITLE
How to get x,y from ./darknet detector test?

### ISSUE THREAD

**Issue by Jaelly** [AlexeyAB/darknet#8092]
If something doesn’t work for you, then show 2 screenshots:
1. screenshots of your issue
![Screenshot from 2021-09-17 16-46-02](https://user-images.githubusercontent.com/82497626/133747618-b25d407f-104e-4eb0-b3ec-734b84d1afd6.png)

2. screenshots with such information
```
Just question,, not error.

./darknet detector test cfg/coco.data cfg/yolov4.cfg yolov4.weights data/dog.jpg
I got object recognition image through above command. Is there a way for me to get the x,y information values of these objects?
I received it from /darknet_ros/bindingboxes from darknet_ros by video, but is it also possible for that image frame?

**Comment by stephanecharette** [comment]
Yes, please see the readme which has all the ways you can run darknet.
Then there is the original C API which will give your the coordinates and many other details.
Then there is the C++ API, the python API, etc., which also gives that information.
Then there are supplementary tools like DarkHelp which also have many different ways to get that information.

**Comment by Jaelly** [comment]
@stephanecharette 
I'm sorry for asking such a simple thing :,(
I'll check it! Thanks.
